reference/technical_information/choose_database.rst

```diff
@@ -0,0 +1,35 @@
+Depending on your data volume (number of products, number of attributes per product, number of basic, localized or scopable attributes, number of locales, number of scopes, etc.) you will have to choose between the following database server configurations:
+
+ * Full SQL database with MySQL
+ * Hybrid SQL storage with MySQL and MongoDB
+
+To make an educated choice you should try to find out how many product values your database will store. For example, for one product:
+
+ * A simple attribute will generate only one product value
+ * A localized attribute will generate as many product values as you have locales enabled
+ * A scopable attribute will generate as many product values as you have channels
+ * A localizable and scopable attribute will generate: (number of enabled locales * number of channels) product values
+
+Here is the complete formula to check if you have, by far, more product values than the recommended threshold that MySQL can manage alone:
+
+.. code-block:: yaml
+
+    N products * (
+        N simple attributes
+        + ( N localized attributes * N enabled locales )
+        + ( N scopable attributes * N existing channels )
+        + ( N scopable AND localizable attributes * N enabled locales * N existing channels )
+    ) > 5 Millions
+
+This formula is often simplified to the following:
```

Comment:
I would not put this simplified version, as it would encourage people to migrate/use MongoDB where they have no need. According to our study, we have around 10% of attributes localizables, 5% of scopables and 2% that are both.
And with the cost to switch to MongoDB (with the complexity and the specific problems like the 64 index limit), I would say that the switch must be done when there's no other choice ;)